Makefile

```diff
@@ -94,13 +94,13 @@ local-ecr-login:
 .PHONY: local-hostconfig
 local-hostconfig:
 	if [ "$$(uname -s)" == "Darwin" ]; then \
-	  sudo ./scripts/happy hosts install; \
+	  sudo -E ./scripts/happy hosts install; \
```

Comment:
required in order to work with my local pyenv configurations, and I see no reason this would disturb others' workflows. `-E` preserves env variables for sudo commands.